docs/tutorial.rst

```diff
@@ -255,23 +255,23 @@ Phasing/unphasing data
 ::
 
   >>> from pyuvdata import UVData
-  >>> import ephem
+  >>> from astropy.time import Time
   >>> UV = UVData()
   >>> miriad_file = 'pyuvdata/data/new.uvA'
   >>> UV.read_miriad(miriad_file)
   >>> print(UV.phase_type)
   drift
 
   # Phase the data to the zenith at first time step
-  >>> UV.phase_to_time(UV.time_array[0])
+  >>> UV.phase_to_time(Time(UV.time_array[0], format='jd'))
```

Comment:
I had to add some optional keywords, but I didn't *have* to change the representation of time. 

The problem was that the old format was super unclear. The docstring just said "time: The time to phase to.". Internally, it was assumed to be a Julian date.

I could have just clarified the docstring that it was a Julian date, but requiring that it was an Astropy time object seemed safer because it's easier to check that it's sensible and it both allows for other time formats (anything Astropy time can handle) and forces the user to think about what they're specifying.